### ISSUE TITLE
ABR: IllegalStateException: Card number not found: 20

### ISSUE THREAD

**Issue by uckelman** [vassalengine/modules#462]
| Bugzilla |  |
|----------|----------|
| ID | BZ3580 |
| Reported | 2011-04-18 11:20:00 +0200 |
| Modified | 2012-09-20 16:48:56 +0200 |
| Product | VASSAL |
| Component | Unknown |
| Version | 3.1.15 |
| Platform | All |
| OS | All |
| Status | RESOLVED |
| Resolution | DUPLICATE |
| Priority | unspecified |
| Severity | normal |
| Milestone | --- |
| Reporter | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Assignee | [Joel Uckelman](mailto:uckelman@nomic.net) |
| CC | b.easton@exemail.com.au |
| Duplicate of | BZ3578 |

| Attachments | |
|-------|-------|
| [errorLog](http://www.vassalengine.org/files/bz/2933) | [Joel Uckelman](mailto:uckelman@nomic.net) 2011-04-18 11:20:00 +0200 text/plain 9639 the errorLog |

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2011-04-18 11:20:40 +0200
<pre>---- Reported by the Automatic Bug Reporter ----

Reported by bonomo@rand.org

Seems to be a problem with the Olympics card now (resolved the prior bug yesterday by starting the recording of the next vlog only after the prior vlog was done, even though a dialog box opened in the prior vlog for the non-phasing player).  Now though, crashing when the module should allow me, the non-phasing player, to pick whether to boycott or compete.</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2012-09-20 16:48:56 +0200
<pre>*** This bug has been marked as a duplicate of bug 3578 ***</pre>
